### ISSUE TITLE
MS_WIN64 need to be add to C(XX)_FLAGS for mingw64

### ISSUE THREAD

**Issue by sanderboer** [tpaviot/pythonocc-core#48]
Hi,

I managed to compile pythonocc for python2, but I had to add MS_WIN64 to the C_FLAGS and CXX_FLAGS :

```
-DCMAKE_C_FLAGS_RELEASE="-O3 -DNDEBUG -DMS_WIN64" \
-DCMAKE_C_FLAGS_MINSIZEREL="-Os -DNDEBUG -DMS_WIN64" \
-DCMAKE_C_FLAGS_DEBUG="-g -DMS_WIN64" \
-DCMAKE_C_FLAGS_RELWITHDEBINFO="-O2 -g -DNDEBUG -DMS_WIN64" \
-DCMAKE_CXX_FLAGS_RELEASE="-O3 -DNDEBUG -DMS_WIN64" \
-DCMAKE_CXX_FLAGS_MINSIZEREL="-Os -DNDEBUG -DMS_WIN64" \
-DCMAKE_CXX_FLAGS_DEBUG="-g -DMS_WIN64" \
-DCMAKE_CXX_FLAGS_RELWITHDEBINFO="-O2 -g -DNDEBUG -DMS_WIN64" \
```

Not an issue, more a tip, I guess.

gr.
S

**Comment by tpaviot** [comment]
Issue reproduced, as well as the suggested fix. Thanks @sanderboer

**Comment by tpaviot** [comment]
Fixed by commit 9aef6c8
